content/en/blog/2021-03-19-a-parrot-trainer-eats-crow.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "A Parrot Trainer Eats Crow"
+slug: "a-parrot-trainer-eats-crow"
+date: "2021-03-19T08:42:21-04:00"
+draft: false
+image_webp: images/blog/parrots.webp
+image: images/blog/parrots.jpg
+author: Rebecca Bilbro
+description : "Models trained on massive datasets using millions of parameters can both have low bias and also be very biased - what can we in the ML community do about this?"
+---
+
+In this post, we'll consider how it is that models trained on massive datasets using millions of parameters can be both "low bias" and also *very biased*, and begin to think through what we in the ML community might be able to do about it.
+
+## Birds on a Wire
+
+In the machine learning community, we are trained to think of size as inversely proportional to bias. We associate small datasets with the problem of underfit, which is to say high bias. We learn that in the face of unfamiliar data, underfit models make poor assumptions that lead to inaccuracies. Likewise, we call models with smaller sets of hyperparameters "weak learners" because their limited complexity limits our ability to reduce bias even as our dataset size grows.
+
+This intuition has driven the ML community towards ever larger datasets and increasingly complex model architectures, and to be sure, towards ever better accuracy scores. Unfortunately (and not unironically), this progression has driven a wedge between the ML definition of "bias" and the more colloquial sense of the word.
+
+
+## Migration Patterns
+
+To understand our situation, it may help to trace back through the pattern of our collective migration towards these more complex models.
+
+> "Is deep learning really necessary to solve most machine learning problems?"
```

Comment:
I appreciate taking this line a lot - it's courageous for the reasons you state in the next para!